modules/compute/gke-node-pool/main.tf

```diff
@@ -33,6 +33,8 @@ locals {
   autoscale_set    = var.autoscaling_total_min_nodes != 0 || var.autoscaling_total_max_nodes != 1000
   static_node_set  = var.static_node_count != null
   initial_node_set = try(var.initial_node_count > 0, false)
+
+  module_unique_id = replace(lower(var.ineternal_filed_to_be_used_by_secret_coven_of_mages_do_not_touch), "/[^a-z0-9]/", "")
```

Comment:
seems to be too strict, aren't `-` also allowed?

Reply:
done